tests/deprecated_api/test_remove_1-8.py

```diff
@@ -146,6 +146,16 @@ def test_v1_8_0_trainer_verbose_evaluate():
         trainer.verbose_evaluate = False
 
 
+@pytest.mark.parametrize("fn_prefix", ["validated", "tested", "predicted"], ids=["validated", "tested", "predicted"])
```

Comment:
What's the benefit of setting `id`s here?

```suggestion
@pytest.mark.parametrize("fn_prefix", ["validated", "tested", "predicted"])
```

Reply:
The benefit is you get to inadvertently test an awesome engineer's considerable peer review prowess when you alter variables and forget to delete the now superfluous ids 😂. Thanks @carmocca !